hxat/settings/test.py

```diff
@@ -59,6 +59,7 @@
 LTI_SECRET_DICT = {
     TEST_COURSE: TEST_COURSE_LTI_SECRET,
 }
+ADMIN_ROLES = {"Administrator", "Instructor"}
```

Comment:
The use of a dict here, rather than a tuple, prompted me to look at the `ADMIN_ROLES` definition in `base.py`. Its outside of the scope of this PR, but it seems like it has an overly complex definition.

Reply:
Ohhh, thats a set.